go.mod

```diff
@@ -48,12 +48,13 @@ require (
 	k8s.io/kubectl v0.18.2
 	sigs.k8s.io/controller-runtime v0.6.0
 	sigs.k8s.io/controller-tools v0.3.0
-	sigs.k8s.io/kubebuilder v1.0.9-0.20200513134826-f07a0146a40b
+	sigs.k8s.io/kubebuilder v0.0.0
 	sigs.k8s.io/yaml v1.2.0
 )
 
 replace (
 	github.com/Azure/go-autorest => github.com/Azure/go-autorest v13.3.2+incompatible // Required by OLM
 	github.com/mattn/go-sqlite3 => github.com/mattn/go-sqlite3 v1.10.0
 	k8s.io/client-go => k8s.io/client-go v0.18.2
+	sigs.k8s.io/kubebuilder => github.com/camilamacedo86/kubebuilder v1.0.9-0.20200528130124-1a29329e09b2
```

Comment:
Hi @estroz, 

Here I bumped the 3.1.0 plugin which will scaffold new fille (.dockerignore) to evaluate the plugins impl when new versions are introduced. (branch: camilamacedo86:test-add-newpluginversion)

All worked fine. However, the PROJECT version is: 

```
domain: testproject.org
layout: go.kubebuilder.io/v3.1.0
repo: plugin-minor
version: 3-alpha
plugins:
  go.operator-sdk.io/v3.1.0: {}
```

See that I did change the plugin version on SDK side, so it should not means that `go.operator-sdk.io/v3.1.0: {}` should be v2.0.0 instead?